### ISSUE TITLE
Connect-NSXServer Auto detect NSX from existing vCenter Connect-viserver session

### ISSUE THREAD

**Issue by gavinbarnard** [vmware/powernsx#75]
Most customers I am pushing PowerNSX to are heavy PowerCLI users and already have established Connect-ViServer starting as a part of their PowerCLI Profile.

excerpt from a script I have that finds NSX on a connected VC Session:
```

function lC_find_nsx
{

if($global:DefaultViServers.Count -eq 0)
{
    lC_err_handler("There are no connected vCenters",$global:DefaultViServers,"Please run Connect-VIServer")
    exit
}

if (!$global:DefaultViServers.Count -gt 1) 
{
    lC_err_handler("More than 1 vCenter connection found",$global:DefaultViServers,"The scope of this script does not currently support multiple vCenter connections")
    exit
}
	try {
		$nsx_ext = (Get-View ExtensionManager).ExtensionList | Where {$_.Key -match "com.vmware.vShieldManager"} 
		[string]$nsx_url = $nsx_ext.Client.Url 
		$nsx_url = $nsx_url.Substring(0,$nsx_url.IndexOf("/",8))
	} catch {
		lC_err_handler($_.Exception.Message,$_.Exception.ItemName,"Unable to retrieve NSX Manager URL, your user may not have permission to read vCenter Extension data")
        exit
	}
    return $nsx_url
}
```

**Comment by nmbradford** [comment]
Thanks!  Thats gold, so maybe allow Connect-NsxServer to be run without a server argument if vi is connected and NSX registration can be retrieved? :)

**Comment by nmbradford** [comment]
Closed now functionality is merged in #175
